HW_1/task06.py

```diff
@@ -0,0 +1,21 @@
+# Задача 6: Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером. 
+# Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. 
+# Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+# *Пример:*
+# 385916 -> yes
+# 123456 -> no
+
+bilet = int(input("Введите номер билета из 6 цифр: "))
+b1 = bilet // 1000
+b2 = bilet % 1000
+sum1 = 0
+sum2 = 0
+while b1 and b2 > 0:
```

Comment:
Раз уж на то пошло, то можно просто
```suggestion
while b1 and b2:
```